sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -578,26 +567,26 @@ def get_next(next_link=None):
         return ItemPaged(get_next, extract_data)
 
     @overload
-    def update_manifest_properties(self, repository, tag_or_digest, properties, **kwargs):
-        # type: (str, str, ArtifactManifestProperties, **Any) -> ArtifactManifestProperties
-        pass
-
-    @overload
-    def update_manifest_properties(self, repository, tag_or_digest, **kwargs):
-        # type: (str, str, **Any) -> ArtifactManifestProperties
-        pass
-
-    @distributed_trace
-    def update_manifest_properties(self, *args, **kwargs):
-        # type: (Union[str, ArtifactManifestProperties], **Any) -> ArtifactManifestProperties
+    def update_manifest_properties(
+        self,
+        repository: str,
+        tag_or_digest: str,
+        properties: ArtifactManifestProperties,
+        *,
+        can_delete: Optional[bool] = None,
+        can_list: Optional[bool] = None,
+        can_read: Optional[bool] = None,
+        can_write: Optional[bool] = None,
+        **kwargs
+    ) -> ArtifactManifestProperties:
         """Set the permission properties for a manifest.
 
         The updatable properties include: `can_delete`, `can_list`, `can_read`, and `can_write`.
 
         :param str repository: Repository the manifest belongs to.
         :param str tag_or_digest: Tag or digest of the manifest.
         :param properties: The property's values to be set. This is a positional-only
-         parameter. Please provide either this or individual keyword parameters.
+            parameter. Please provide either this or individual keyword parameters.
```

Comment:
If this is supposed to be _either_ properties _or_ keyword parameters, then the keyword parameters shouldn't be in this overload.